defaults/sdwan.yaml

```diff
@@ -316,3 +316,7 @@ defaults:
         aaa:
           auth_order:
             - local
+    centralized_policies.definitions:
+      data_policy:
+        cflowd:
+          protocol: "ipv4"
```

Comment:
we should also add defaults for tos: false and remarked_dscp: false (when you configure cflowd in UI, by default they are set to false)